### DISCUSSION TITLE
Prowler .csv header question re: ";COMPLIANCE;CATEGORIES;"

### DISCUSSION THREAD

**Discussion by edschreibman** [prowler-cloud/prowler#2473] [Q&A] [answered]
Running v3.5.3;
" prowler -M csv   --verbose"
Noticed the .csv header results in 38 fields, was expecting 37.
Looking at the header, should ";COMPLIANCE;CATEGORIES;"  be ";COMPLIANCE_CATEGORIES;"  ?   
It looks like it is creating a new column in error.

**Comment by jfagoagas** [comment]
Hi @edschreibman, I'm going to move this discussion to an issue.

Thanks for using Prowler 🚀

**Comment by MrCloudSec** [comment, marked as answer]
Hi @edschreibman, the Compliance and Categories columns are two different attributes:

- The Compliance column contains all the compliance's requirements applicable to each finding
- The Category column contains the categories applicable to the finding that can be seen in each check's metadata.
